chartify/_core/colors.py

```diff
@@ -43,8 +43,7 @@ def to_yaml(self, filename):
     def from_yaml(self, filename):
         """Load colors from yaml file"""
         # Note: We assume that the contents of the config file are trusted
-        # TODO: Change this file format to be plain yaml and use SafeLoader
-        return yaml.load(open(filename), Loader=yaml.UnsafeLoader)
+        return yaml.load(open(filename), Loader=yaml.SafeLoader)
```

Comment:
It doesn't seem to work, a test fails with this change. See e.g. https://death.andgravity.com/yaml-unknown-tag about how to perhaps work around this.